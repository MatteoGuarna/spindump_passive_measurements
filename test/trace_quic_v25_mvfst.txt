A test with Mvfst QUIC implementation server and Quant client.

There are four sessions:

- regular + h3 + full start
- regular + h3 + 0rtt 
- regular + h3
- key update + h3
- quantum + h3

The commands used were:

   (rm /tmp/quant*;
   ./bin/client -3 -i en0 https://fb.mvfst.net:443/5000 ;
   ./Bin/client -3 -i en0 https://fb.mvfst.net:443/15000 ; 
   ./Bin/client -3 -i en0 https://fb.mvfst.net:443/15000 ; 
   ./bin/client -3 -u -i en0 https://fb.mvfst.net:443/10000;
   ./bin/client -3 -m -i en0 https://fb.mvfst.net:443/20000)

