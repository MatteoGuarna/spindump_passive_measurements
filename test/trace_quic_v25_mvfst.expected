QUIC 3.15.16.10 <-> 100.72.46.172 null-null (443:54135) at 1580828346265736 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xffffffff,no spin"
QUIC 3.15.16.10 <-> 100.72.46.172 null-null (443:54135) at 1580828346265739 packet starting initiator length 288 packets 2 0 bytes 576 0 bandwidth 576 0 note "No rsp,v.0xffffffff,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 b2e40a48-c305d832 (59003:443) at 1580828355864893 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 b2e40a48-de81ceeb11c305 (59003:443) at 1580828355974680 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 b2e40a48-de81ceeb11c305 (59003:443) at 1580828355974680 measurement starting right 109787 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v.0x00000000,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 null-de81ceeb11c305 (59003:443) at 1580828355976975 measurement starting left 2295 packets 1 1 bytes 1228 68 bandwidth 1228 68 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 null-084000025954b8f1 (59003:443) at 1580828356082550 change starting packets 2 1 bytes 2456 68 bandwidth 2456 68 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 null-084000025954b8f1 (59003:443) at 1580828356082550 measurement up right 105575 packets 2 1 bytes 2456 68 bandwidth 2456 68 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356082553 packet up responder length 159 packets 2 3 bytes 2456 298 bandwidth 2456 298 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356083376 packet up initiator length 1228 packets 3 3 bytes 3684 298 bandwidth 3684 298 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356183525 packet up responder length 159 packets 3 4 bytes 3684 457 bandwidth 3684 457 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356183899 packet up initiator length 72 packets 4 4 bytes 3756 457 bandwidth 3756 457 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356197330 packet up responder length 72 packets 4 5 bytes 3756 529 bandwidth 3756 529 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356197335 packet up responder length 194 packets 4 6 bytes 3756 723 bandwidth 3756 723 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356197541 packet up responder length 1280 packets 4 7 bytes 3756 2003 bandwidth 3756 2003 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356197545 packet up responder length 1280 packets 4 8 bytes 3756 3283 bandwidth 3756 3283 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356197888 packet up responder length 638 packets 4 9 bytes 3756 3921 bandwidth 3756 3921 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356198454 packet up initiator length 72 packets 5 10 bytes 3828 4006 bandwidth 3828 4006 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 4be84522-084000025954b8f1 (59003:443) at 1580828356200478 packet up initiator length 1283 packets 6 10 bytes 5111 4006 bandwidth 5111 4006 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f88afa-d9590949059916dd09c803cc6cf2 (60618:443) at 1580828356607523 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f88afa-084000030303af7a (60618:443) at 1580828356746490 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f88afa-084000030303af7a (60618:443) at 1580828356746490 measurement up right 138967 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356746496 packet up responder length 199 packets 1 2 bytes 1228 270 bandwidth 1228 270 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356746497 packet up responder length 239 packets 1 3 bytes 1228 509 bandwidth 1228 509 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356749646 packet up initiator length 128 packets 2 14 bytes 1356 12955 bandwidth 1356 12955 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356750448 packet up initiator length 1283 packets 3 14 bytes 2639 12955 bandwidth 2639 12955 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356816089 packet up responder length 199 packets 4 15 bytes 2698 13154 bandwidth 2698 13154 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356816094 packet up responder length 239 packets 4 16 bytes 2698 13393 bandwidth 2698 13393 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f88afa55-084000030303af7a (60618:443) at 1580828356816809 packet up initiator length 71 packets 5 16 bytes 2769 13393 bandwidth 2769 13393 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f7967a-2d1bb70f124c26b175f116 (55766:443) at 1580828357163895 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f7967a-084000001273b256 (55766:443) at 1580828357270671 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04f7967a-084000001273b256 (55766:443) at 1580828357270671 measurement up right 106776 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357270677 packet up responder length 199 packets 1 2 bytes 1228 270 bandwidth 1228 270 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357270678 packet up responder length 239 packets 1 3 bytes 1228 509 bandwidth 1228 509 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357273707 packet up initiator length 128 packets 2 14 bytes 1356 12955 bandwidth 1356 12955 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357274400 packet up initiator length 1283 packets 3 14 bytes 2639 12955 bandwidth 2639 12955 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357371058 packet up responder length 199 packets 4 15 bytes 2698 13154 bandwidth 2698 13154 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357371064 packet up responder length 239 packets 4 16 bytes 2698 13393 bandwidth 2698 13393 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357371790 packet up initiator length 71 packets 5 16 bytes 2769 13393 bandwidth 2769 13393 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 f7967acd-084000001273b256 (55766:443) at 1580828357383830 measurement up full (initiator) 629 packets 7 21 bytes 2889 17257 bandwidth 2889 17257 note "V25,0-RTT,no R-spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04d14248-f991a12fae5dedc0 (59342:443) at 1580828357688566 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04d14248-0840000281e8c6b8 (59342:443) at 1580828357795235 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 04d14248-0840000281e8c6b8 (59342:443) at 1580828357795235 measurement up right 106669 packets 1 0 bytes 1228 0 bandwidth 1228 0 note "No rsp,v25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357795241 packet up responder length 199 packets 1 2 bytes 1228 270 bandwidth 1228 270 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357795242 packet up responder length 239 packets 1 3 bytes 1228 509 bandwidth 1228 509 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357798290 packet up initiator length 128 packets 2 13 bytes 1356 11137 bandwidth 1356 11137 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357799063 packet up initiator length 1283 packets 3 13 bytes 2639 11137 bandwidth 2639 11137 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357895785 packet up responder length 199 packets 4 14 bytes 2698 11336 bandwidth 2698 11336 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357895789 packet up responder length 239 packets 4 15 bytes 2698 11575 bandwidth 2698 11575 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357896334 packet up initiator length 71 packets 5 15 bytes 2769 11575 bandwidth 2769 11575 note "V25,0-RTT,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 d142488b-0840000281e8c6b8 (59342:443) at 1580828357964534 measurement up full (initiator) 58505 packets 6 18 bytes 2830 12082 bandwidth 2830 12082 note "V25,0-RTT,no R-spin"
QUIC 100.72.46.172 <-> 3.15.16.10 0449a00b-9c42b0393e66b882cdd5af439e169cdec2a175 (64547:443) at 1580828358612533 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 0449a00b-9c42b0393e66b882cdd5af439e169cdec2a175 (64547:443) at 1580828358612533 packet starting initiator length 753 packets 2 0 bytes 2033 0 bandwidth 2033 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 0449a00b-08400000c879dd90 (64547:443) at 1580828358722119 change starting packets 2 0 bytes 2033 0 bandwidth 2033 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 0449a00b-08400000c879dd90 (64547:443) at 1580828358722119 measurement up right 109586 packets 2 0 bytes 2033 0 bandwidth 2033 0 note "No rsp,v25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358722125 packet up responder length 71 packets 2 2 bytes 2033 142 bandwidth 2033 142 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358722127 packet up responder length 199 packets 2 3 bytes 2033 341 bandwidth 2033 341 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358722128 packet up responder length 239 packets 2 4 bytes 2033 580 bandwidth 2033 580 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358725157 packet up initiator length 128 packets 3 14 bytes 2161 12190 bandwidth 2161 12190 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358725832 packet up initiator length 1283 packets 4 14 bytes 3444 12190 bandwidth 3444 12190 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358820957 packet up responder length 199 packets 6 16 bytes 3565 13225 bandwidth 3565 13225 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358820963 packet up responder length 239 packets 6 17 bytes 3565 13464 bandwidth 3565 13464 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358821680 packet up initiator length 71 packets 7 17 bytes 3636 13464 bandwidth 3636 13464 note "V25,no spin"
QUIC 100.72.46.172 <-> 3.15.16.10 49a00bf6-08400000c879dd90 (64547:443) at 1580828358837641 measurement up full (initiator) 617 packets 8 22 bytes 3697 18639 bandwidth 3697 18639 note "V25,no R-spin"
QUIC 3.15.16.10 <-> 100.72.46.172 null-null (443:54135) at 1580828366882841 packet starting initiator length 68 packets 3 0 bytes 644 0 bandwidth 576 0 note "No rsp,v.0xffffffff,no spin"
QUIC 3.15.16.10 <-> 100.72.46.172 null-null (443:54135) at delete starting packets 3 0 bytes 644 0 bandwidth 576 0 note "No rsp,v.0xffffffff,no spin"
